### ISSUE TITLE
[Question] Is this training log normal or not?

### ISSUE THREAD

**Issue by hungnvk54** [wenet-e2e/wespeaker#425]
Hi,

First of all, I want to thank for your contribution. 

Today, I use your example to retrain voxceleb/Resnet34. Dataset is default vox1, vox2 (download via your default script utils) and my private 800 ids (I mixed my private data with vox2 data).

After training 1 epoch, the training log output is:

![Image](https://github.com/user-attachments/assets/957cb74c-be68-4039-9bf0-c27e8429d7d8)

The Acc is only 0.015 - 0.04 for the first epoch. Is this normal or not. 

(I want to ask this question because the training time is to long (one week), wait util finish tranining is not good idea)

Thank you in advance.

**Comment by cdliang11** [comment]
@hungnvk54 I think it is normal. The picture shows the training early, only 100 batches have passed, and no complete epoch has passed.

**Comment by hungnvk54** [comment]
I'm using shard datatype. So batch size here is calculated by number of audio file or by number of shard?

**Comment by cdliang11** [comment]
number of audio file

**Comment by hungnvk54** [comment]
@cdliang11 Thank for your response

If it is audio file, It mean we need 1.5days/epoch. So it is too long.

I'm traning on server with two GPU RTX 2080Ti. Is it normal?

And which server the pretrained model?

**Comment by cdliang11** [comment]
1. it's abnormal.
2. pretrained model: https://github.com/wenet-e2e/wespeaker/blob/master/docs/pretrained.md

**Comment by hungnvk54** [comment]
Hi @cdliang11 , Can you share the configure for traning Resnet221. In the voxceleb/v2, There are no example configure for Resnet211. I found the Resnet211_LM in huggingface, but it seems that it is configure for LM, the postprocess of from the Resnet221?

**Comment by cdliang11** [comment]
I checked `voxceleb_resnet221_LM.yaml` on huggingface. But, I found that it should be `voxceleb_resnet221.yaml`. Sorry, it might be a typo. You can use it directly.   https://huggingface.co/Wespeaker/wespeaker-voxceleb-resnet221-LM/blob/main/voxceleb_resnet221_LM.yaml

**Comment by MM-WW55** [comment]
> [@hungnvk54](https://github.com/hungnvk54) I think it is normal. The picture shows the training early, only 100 batches have passed, and no complete epoch has passed.

<img width="798" height="300" alt="Image" src="https://github.com/user-attachments/assets/e88f4568-2d23-4ff5-af6e-db3fce984220" />

Hi @hungnvk54 I have a question about my custom model training. I'm in the first epoch and while the loss is decreasing, the accuracy is still 0. Is this normal behavior for the beginning of training, or could this indicate a problem?
Thanks for your help!
